src/Receipt/DonationDetailsGroup/Details/Amount.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Give\Receipt\DonationDetailsGroup\Details;
+
+use Give\Receipt\Detail;
+use function give_get_payment_meta as getDonationMetaData;
+use function give_format_amount as formatAmount;
+use function give_currency_filter as filterCurrency;
+
+/**
+ * Class Amount
+ *
+ * @since 2.7.0
+ * @package Give\Receipt\DonationDetailsGroup\Details
+ */
+class Amount extends Detail {
+	/**
+	 * @inheritDoc
+	 */
+	public function getLabel() {
+		return __( 'DONATION AMOUNT', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.